packages/front-end/components/Layout/Layout.tsx

```diff
@@ -122,6 +122,22 @@ const navlinks: SidebarLinkProps[] = [
       },
     ],
   },
+  {
+    name: "Integrations",
+    href: "/integrations",
+    Icon: BsPlug,
+    path: /^(integrations)/,
+    autoClose: true,
+    feature: "slack-integration",
+    subLinks: [
+      {
+        name: "Slack",
+        href: "/integrations/slack",
+        path: /^integrations\/slack/,
+        feature: "slack-integration",
```

Comment:
Also added support to conditionally turn on/off sub links. So when we're ready to release Slack integrations, we need to remove this from both the child and parent link. This support allows us to introduce new integrations without hiding the whole parent.